### ISSUE TITLE
Deploy Application

### ISSUE THREAD

**Issue by Ctoic** [Ctoic/Hostel-Management-System-Using-Flask#37]
Can we deploy this flask application for free ?

**Comment by Prathameshw4** [comment]
Yes, it is possible to deploy a Flask application for free, especially for personal projects, learning purposes, or small-scale applications. Several platforms offer free tiers or trial periods that can be utilized for this purpose.
Platforms for Free Flask Deployment:
Render:
Offers a free tier for web services, allowing deployment of Flask applications with limitations on resources like CPU, memory, and bandwidth.
Railway:
Provides a free trial with a certain amount of credit, which can effectively allow for free deployment of minor Flask projects.
PythonAnywhere:
Offers a free account with limitations suitable for testing and sharing small Flask applications.
AWS Free Tier:
Provides a 12-month free tier for various services, including EC2 instances, which can be used to host a Flask application, although it requires more manual setup.
Google Cloud Free Tier:
Similar to AWS, Google Cloud offers a free tier for services like App Engine or Cloud Run, which can be used to deploy Flask applications.

**Comment by Ctoic** [comment]
Go ahead @PrathameshWankhede4  if you wanna give it a try.
